Examples/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(PcapPlusPlusExamples)
+
+set(CMAKE_PROJECT_HOMEPAGE_URL "https://pcapplusplus.github.io/")
+
+if(TARGET Pcap++)
+  # Are we building from the PcapPlusPlus project ??
+  message(STATUS "Building Example from PcapPlusPlus project")
+  # Alias the project to make it compatible with imported package
+  add_library(PcapPlusPlus::Pcap++ ALIAS Pcap++)
+else()
+  find_package(PcapPlusPlus CONFIG REQUIRED)
+  message(STATUS "Using Pcap++ ${PcapPlusPlus_VERSION}")
+  message(STATUS "Include dir: ${PcapPlusPlus_INCLUDE_DIR}")
+endif()
```

Comment:
I'm not sure if this is needed. The examples are part of the PcapPlusPlus repo and are always built as part of the project...

Reply:
This test that we can build Examples after we have installed Pcap++

Reply:
ok, let's keep it as is 👍